docs/get-started/getting-started-alluxio.md

```diff
@@ -419,4 +419,93 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio metrics
+### Monitor Alluxio metrics based on Alluxio Master Web
+Users can view the Alluxio metrics in the Web interface of Alluxio leading master:       
+http://<leading_master_host>:19999/metrics   
+For more details, please refer to section 3.1 of Alluxio doc: [Master Web UI Metrics](https://docs.alluxio.io/os/user/stable/en/operation/Metrics-System.html#default-http-json-sink)      
+The Alluxio Web UI is not available by default on databricks, the following provides a method to view the Web UI by SSH tunnel via jump server.   
+First forward the Alluxio port 19999 to a new port on a jump server, then forward the new port on the jump server to a local port, finally access the local port in the browser.   
+For example:   
+- Forward the port 29999 on jump server to local port 39999 on your own machine.   
```

Comment:
can't you just forward directly to the machine you are on vs jump server as long as it has ssh access?

Reply:
Ideally, executing the following one command on the machine you are on is OK:

```
ssh -L  <local_port>:Alluxio-master-IP:19999 user@jump-server
```
The above command means forward port 19999 on Alluxio-master to <local_port> via jump-server

But, the ssh port on Alluxio-master is 2200 instead of the default value.
I do not know how to specify the `private_key_file_path` and `-p 2200` in one command.

So, we use the workaround:
1. Forward 19999 to jump-server: 29999
2. Forward jump-server: 29999 to local-pc:39999

Reply:
so what is the definition of a jump server here?   Where does user get a jump server?   Is this a databricks specific thing?